packages/client/src/client/components/TabBar/TabBar.tsx

```diff
@@ -36,7 +36,10 @@ export const TabBar = <T extends string>({
   doNotShowDropdown,
 }: TabBarProps<T>) => (
   <Background>
-    <LeftSection doNotShowDropdown={items.length < 2 || !!doNotShowDropdown}>
+    <LeftSection
+      data-testid="tab-bar"
```

Comment:
should be able to remove the `data-testid` on the individual tabs now?
when making changes like the above, we need to cast around to see if that affects things, and whether we can clean up code in the near vicinity ..

Reply:
done .. but a bunch of knock-on effects in unit tests and FX specs .. shows it was a bit brittle, relying on tabs always being there!